db/migrate/20170519123223_add_request_uuid_to_audits.rb

```diff
@@ -0,0 +1,10 @@
+class AddRequestUuidToAudits < ActiveRecord::Migration
+  def self.up
+    add_column :audits, :request_uuid, :string
```

Comment:
Yes, it is required. No, the audits gem just says to run a particular rails generate command after each upgrade and when I didn't it crashed (on boot, I think). I haven't checked but I'm pretty sure; I'll do that and we should have tests for it?